<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompanyWindow</class>
 <widget class="QMainWindow" name="AddCompanyWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KB Roadways - Add Company</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../__Assets/Logos/CompanyLogo.png</normaloff>../__Assets/Logos/CompanyLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCompanyName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCompanyAddress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Address</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>260</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../__Assets/Logos/CompanyLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCompanyName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teCompanyAddress">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblContactNo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contact No.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leContactNo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>461</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblNotes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Notes (If any)</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="teNotes">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCreateCompany">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Create Company</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewCompanies">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>500</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>View Companies</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leCompanyName</tabstop>
  <tabstop>teCompanyAddress</tabstop>
  <tabstop>leContactNo</tabstop>
  <tabstop>teNotes</tabstop>
  <tabstop>btnCreateCompany</tabstop>
  <tabstop>btnReset</tabstop>
  <tabstop>btnViewCompanies</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
